src/openrct2/paint/Supports.cpp

```diff
@@ -634,7 +644,7 @@ bool WoodenBSupportsPaintSetup(
         imageTemplate = ImageId().WithTransparency(FilterPaletteID::PaletteDarken1);
     }
 
-    uint16_t baseHeight = Ceil2(session.Support.height, 16);
+    auto baseHeight = Floor2(session.Support.height + 15, 16);
```

Comment:
Most of the changes on commit one were stylistic, but this one was an actual change, can you elaborate on it?

Reply:
When you run through all values, it actually comes out as exactly the same value:

For `Floor2(session.Support.height + 15, 16);`:
orig. height|parameter 1|result
---|---|---
0|15|0
1|16|16
2|17|16
16|31|16
17|32|32
18|33|32

For `Ceil2(session.Support.height, 16);`
orig. height|result
---|---
0|0
1|16
2|16
16|16
17|32
18|32

Reply:
I am curious what the benefit of this is

Reply:
We had two functions that were 99% the same, but using different names, comments, etc. I first eliminated the difference, and then moved some parts that were the same into functions of their own. The result is clearer code and less duplicate code. That’s the benefit.